### ISSUE TITLE
Size of saved model too big (~ 2 gb)

### ISSUE THREAD

**Issue by rlrahulkanojia** [microsoft/multiview-human-pose-estimation-pytorch#27]
the final model saved on a mobile net backbone is of 2 GB. Can someone. please help me where I might be wrong. I have not made any significant changes in the code.

**Comment by CHUNYUWANG** [comment]
The model is big because of the fusion layer which is implemented by fully connected layer.

**Comment by rlrahulkanojia** [comment]
Yes, I found it after a detailed analysis. But the role of optional aggregations layers is not clear to me. It would be very helpful if you could provide any clarification.

**Comment by CHUNYUWANG** [comment]
We will fuse features across different views if the optional aggregation parameter is true. You can take a look at the code [https://github.com/microsoft/multiview-human-pose-estimation-pytorch/blob/7107c1b0ffa3cb9f5783ebdd5726276e44684afc/lib/models/multiview_pose_resnet.py#L87](here)
